app/controllers/contributions_controller.rb

```diff
@@ -15,6 +15,26 @@ def index
     end
   end
 
+  def show
+    contribution = Listing.find(params[:id])
+
+    render(
+      :show,
+      locals: {
+        contribution: contribution,
+      }
+    )
+  end
+
+  def claim_ask_form
+    contribution_id = params[:id]
+    render locals: { contribution_id: contribution_id }
+  end
+
+  def claim_ask
+    redirect_to contribution_path(params[:id])
```

Comment:
This non-RESTful route handles a `POST` verb, but it doesn't look like it creates any resources. Was this intended to create a `Match`?

Reply:
It is. We just wanted to get what we have so far up for review.

Reply:
Moved this method to `claims_controller` which is a nested resource of `contributions` And creating a match there.